include/mrdox/Generator.hpp

```diff
@@ -66,7 +66,8 @@ struct Generator
         llvm::StringRef rootPath,
         Corpus const& corpus,
         Config const& config,
-        Reporter& R) const;
+        Reporter& R,
+        const std::string & fileName) const;
```

Comment:
The `Reporter` always comes last in parameter lists but why are you changing the API of `Generator`?